teams/vsp/teams/tools/backend/pact/readme.md

```diff
@@ -1 +1,420 @@
+
+#### Idenfity Expected Responses
+
+In order to determine an endpoints expected response, take a look at the defined response in the broker. A helpful tip if you're having issues is to look at vets-api request specs to determine provider state implementation details and expected responses.
+
+#### Naming Guidelines
+Provider states are currently defined in the service_consumers/provider_states_for/*.rb directory. Provider States will need to **follow a strict naming protocol** and are to be categorized by consumer name per pact. e.g. Search, Users, HCA, etc. See consumer column [here](https://vagov-pact-broker.herokuapp.com/) for examples. The consumer name in your defined `provider_state_for` block **must** match the name of the consumer configured in your consumer project for the verification task to correctly find the provider states. See [Search Example](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/service_consumers/provider_states_for/search.rb#L3)
+
+The [provider state block](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/service_consumers/provider_states_for/search.rb#L4) will need to match the corresponding definition defined in the pact as well. In the [search example](https://vagov-pact-broker.herokuapp.com/pacts/provider/VA.gov%20API/consumer/Search/latest) a "multiple matching results exist" interaction is defined and a provider state matching this defintion will need to be defined on the [backend](https://github.com/department-of-veterans-affairs/vets-api/pull/4612/files#diff-102a9104474b45510528e3e28a8071c0R4). 
+
+#### Authorization
+If an authorized vets-api user is needed for a request to an endpoint, a helper method can be used to stub the session. Call the `build_user_and_stub_session`  [method](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/stub_session.rb#L3) if your endpoint requires an authorized (signed in) user. See this [forms example](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/service_consumers/provider_states_for/hca.rb). 
+
+#### VCR
+Many of the vets-api endpoints call out to external services. To mock external service calls, the pact helper has VCR configured for usage. VCR cassettes can be used to mock external service responses for many of the vets-api third party service calls. The defined provider state in the search example from above makes use of a VCR cassette.
+
+### Configure the pact_uri/broker_url
+
+To work with one pact in the broker vs all pacts, point the `pact_uri` in the `pact_helper.rb` file to a specific contract in the broker (see [here](https://github.com/department-of-veterans-affairs/vets-api/pull/4612/files#diff-79dfb3765c26716457a005b9d343a160R38)), otherwise the rake task will run all the pacts pushed to the [heroku broker](https://vagov-pact-broker.herokuapp.com/).
```

Comment:
Should this section mention using [`rake pact:verify:at`](https://github.com/pact-foundation/pact-ruby/wiki/Verifying-pacts#verifying-a-pact-at-any-url-using-rake-pactverifyat)?